test/v3/routes/tracks_test.rb

```diff
@@ -27,4 +27,28 @@ def test_tracks
     blueberry = JSON.parse(last_response.body)["track"]
     Approvals.verify(blueberry, name: 'v3_track_unimplemented')
   end
+
+  def test_readme
+    get '/tracks/fake/three/readme'
+
+    options = { format: :json, name: 'v3_valid_readme' }
+    Approvals.verify(last_response.body, options)
+  end
```

Comment:
You mean in v1/routes/ and also v3/routes? Or did you get a weird merge that left both readme routes under v3?